### DISCUSSION TITLE
Check channel content

### DISCUSSION THREAD

**Discussion by ENG-Rabbit** [aiogram/aiogram#478] [Q&A] [answered]
I wanted to know how to check the contents of a channel with aiogram and extract information from them?

**Comment by evgfilim1** [comment, marked as answer]
There's no way to fetch contents of the channel without some hacks and without adding a bot as an admin of the channel. So, if you have possibility to make the bot admin, you can:

1) Fetch incoming updates via `@dp.channel_post_handler`
2) (hacky way) Try forwarding every message from the channel (by bruteforcing IDs and fetching forwarded messages' info, see [forwardMessage](https://core.telegram.org/bots/api#forwardmessage) docs)

Otherwise you have to use so-called userbots (bots that act from a user account)
